core/src/main/java/com/google/zxing/DecodeHintType.java

```diff
@@ -98,6 +98,12 @@
    */
   ALLOWED_EAN_EXTENSIONS(int[].class),
 
+  /**
+   * If true, also tries to decode as negative image. All configured decoders are simply called a
```

Comment:
Can we call this 'inverted' in the docs and name for consistency? no big deal.

Reply:
BinaryBitmap.invert() is actually the old method. I added the new method BitMatrix.flip() and changed the method call, but forgot to remove BinaryBitmap.invert(). I'll do that now.

I renamed negative to inverted. I called the hint ALSO_INVERTED, which should reflect the meaning well (but I'm open for better ideas) while POSITIVE_AND_INVERTED sounds ugly to me.